dactyl-config.yml

```
@@ -370,6 +370,10 @@ pages:
         targets:
             - ja
 
+    -   md: privacy-policy.md
+        targets:
+            - en
```

Gotta add it to both builds since we don't have a dedicated Japanese version of this page yet.

```suggestion
            - en
            - ja
```